integration/experiment/ffnet/README.md

```diff
@@ -0,0 +1,243 @@
+Feed-Forward Net-Based Experiments
+----------------------------------
+
+The scripts in this directory are used to run and analyze application
+runs that use the ffnet agent with a specified path to neural net and
+frequency recommendation maps, optionally with a specified perf energy bias.
+
+## Base Experiment Module
+
+#### `ffnet.py`:
+
+  Contains the helper function for launching an ffnet experiment.
```

Comment:
A minor suggestion for consistency with other docs and geopm_agent_ffnet.7.rst (note: this applies to other instances in the file):

```suggestion
  Contains the helper function for launching an FFNet agent experiment.
```